ark/utils/spatial_analysis_utils.py

```diff
@@ -172,12 +186,11 @@ def compute_close_cell_num(dist_mat, dist_lim, num, analysis_type,
     # iterating k from [j, end] cuts out 1/2 the steps (while symmetric)
     for j, m1n in enumerate(mark1_num):
         for k, m2n in enumerate(mark1_num[j:], j):
-            close_num[j, k] = np.sum(
-                dist_mat_bin[np.ix_(
-                    np.asarray(mark1poslabels[j] - 1, dtype='int'),
-                    np.asarray(mark1poslabels[k] - 1, dtype='int')
-                )]
-            )
+            close_num[j, k] = np.sum(dist_mat_bin.loc[
+                list(mark1poslabels[j]),
+                list(mark1poslabels[k])
```

Comment:
is `mark1poslabels[j]` not a list already?

Reply:
It's actually a pandas series, which breaks the indexing if you use it plainly.

Reply:
😮 gotcha.  I feel like `mark1poslabels[j].values` is viable too then.  idk if one is better than the other tho.  code what ur heart desires.

Reply:
I think .values definitely looks better 😊.